### ISSUE TITLE
Printing command line

### ISSUE THREAD

**Issue by iliobracciali** [sumatrapdfreader/sumatrapdf#1797]
Hello, I want to print a pdf file using the command line. My file has a A3 format page, but my printer support only A4 page.
The option fit or shrink does not work.
The command line I use is:
Sumatrapdf.exe  -print-to-default -print-settings "shrink"  MyFile.pdf
or
Sumatrapdf.exe  -print-to-default -print-settings "fit"  MyFile.pdf

In any case the result is the same, my pdf file is printed but is not reduced to the A4 page dimansion

**Comment by GitHubRulesOK** [comment]
shrink and fit are "preferences" so may be over-ridden by the default set preference so check that settings,txt is also set to "shrink" that then is the "signal" sent to the default printer setup so check the target device is defaulting to A4 not A3 AND MOST IMPORTANTLY the default page size for the DEFAULT printer is also A4 since one of those would appear to be set to a larger size.
you could try

Sumatrapdf.exe -print-to-default -print-settings `"shrink,paper=A4"` MyFile.pdf

In short SumatraPDF approximatly streams the page at its native size (including using larger amounts of memory) to the default SYSTEM print drivers with indications of desired scale and target.

**Comment by GitHubRulesOK** [comment]
@iliobracciali 
Without further feedback I am inclined to close as either resolved (it should be working) or duplicated by similar  user issue  such as https://github.com/sumatrapdfreader/sumatrapdf/issues/1803

**Comment by GitHubRulesOK** [comment]
I am closing as throughout testing numerous other user issues there is normally no problem in resizing a3 documents down to A4 printer output for example the most basic test
SumatraPDF.exe -print-to "Microsoft print to pdf" -print-settings "paper=A4" A3Landscape.pdf
will by default produce A4Portrait.pdf

Only interference by other Windows settings or driver settings may/can/will result in different outputs

For reduction from A3 to A4 the most usable setting is as above
SumatraPDF.exe -print-to-default -print-settings "shrink,paper=A4" A3.pdf
